AutomatedTesting/Gem/PythonTests/Atom/atom_utils/material_canvas_utils.py

```diff
@@ -0,0 +1,20 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import azlmbr.bus as bus
+import azlmbr.atomtools as atomtools
+
+
+def set_pane_visibility(pane_name, value):
```

Comment:
can we have type hints for this function? is value a boolean? looks like it is. maybe use visible for the argument name here to go with the boolean type.
    void AtomToolsMainWindow::SetDockWidgetVisible(const AZStd::string& name, bool visible)
also is pane_name restricted to a defined list of options?

Reply:
this is going to end up getting removed and flattened into a shared set of functions that both MaterialEditor and MaterialCanvas utilize since they both make use of atomtools

this was simply copied from the existing one (which lacks all of the requests being made here as well)